### PR TITLE
background + wedges

### PR DESCRIPTION
**Pull request by Jorolf**
image

### CODE REVIEW

**Comment by sr229** [comment]
Don't forget to add tiles too on the TestCase btw. Use the grid layouts I used on the design.

File: Lovewing.Game/Screens/LovewingScreen.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2007-2017 Clara.
+// Licensed under the MIT License
+
+using osu.Framework.Screens;
+
+namespace Lovewing.Game.Screens
```
Comment (Jorolf) [inline Lovewing.Game/Screens/LovewingScreen.cs]: this basically uses the Screen concept of osu-framework, you can see it in osu

**Comment by Jorolf** [comment]
tiles are for another pr ;)

**Comment by Jorolf** [comment]
and about the blur thing:
idk how to do that and I've never used shaders :/

**Comment by sr229** [comment]
Feel free to consult other developers for shaders :wink:

**Comment by sr229** [comment]
The WedgeButtons has to always cast a soft shadow behind it btw

**Comment by apap04** [comment]
Did you kindly fix the issues?

**Comment by sr229** [comment]
@apap04 I swear to god if you make this a meme I'll rip your dick off

**Comment by Jorolf** [comment]
I kindly added a comment

**Comment by apap04** [comment]
Will you kindly cut my dick off? @sr229

**Comment by sr229** [comment]
goddammit

**Pull request merged**